ui/packages/catalog-ui-search/src/main/webapp/component/available-types/available-types.view.js

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
I believe this component can be deleted now that we're using the property view to construct the available types dropdown.

Reply:
confirmed, and it is now removed